Client/Frontend/Browser/Playlist/Playlist.xcdatamodeld/Playlist.xcdatamodel/contents

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Why does Playlist require an entirely separate CoreData store instead of using the store we already have?